core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -405,18 +405,28 @@
       </div>
     </script>
 
-    <div class="container-fluid oppia-editor-page-container" ng-joy-ride="tutorialInProgress" config="EDITOR_TUTORIAL_OPTIONS" on-finish="onFinishTutorial()" on-skip="onSkipTutorial()">
-      <div class="row" ng-if="explorationRightsService.isCloned()">
-        <div class="col-lg-12 col-md-12 col-sm-12">
-          <div class="oppia-align-center alert alert-warning" style="padding: 2px; width: 90%;">
-            <strong>Note:</strong> This is a private, unpublishable copy of a
-            <a ng-href="<[getExplorationUrl(explorationRightsService.clonedFrom())]>" target="_blank">public exploration</a>.
+    <div ng-joy-ride="tutorialInProgress"
+         config="EDITOR_TUTORIAL_OPTIONS"
+         on-finish="onFinishTutorial()"
+         on-skip="onSkipTutorial()">
+      <div class="container-fluid">
+        <div class="row" ng-if="explorationRightsService.isCloned()">
+          <div class="col-lg-12 col-md-12 col-sm-12">
+            <div class="oppia-align-center alert alert-warning" style="padding: 2px; width: 90%;">
+              <strong>Note:</strong> This is a private, unpublishable copy of a
+              <a ng-href="<[getExplorationUrl(explorationRightsService.clonedFrom())]>" target="_blank">public exploration</a>.
+            </div>
           </div>
         </div>
       </div>
 
       <div ng-show="getTabStatuses().active === 'main'">
-        {% include 'pages/exploration_editor/editor_tab/editor_tab.html' %}
+        <div ng-show="getEditorMode() === 'simple'">
```

Comment:
Prefer to have methods inside of ``EditorModeService`` to do this comparison, e.g. ``EditorModeService.isSimple()``